config/locales/letters_en.yml

```diff
@@ -104,52 +110,56 @@ en-GB:
-        <p>We have received the additional information requested for your application for help with fees. However, there’s a problem with the documents you have sent.</p>
-        <p>(PLEASE EXPLAIN WHY EVIDENCE CAN'T BE ACCEPTED, FOR EXAMPLE: BANK STATMENTS ARE OUT OF DATE</p>
-        <p> Unfortunately we’re unable to process your application.</p>
-        <p> If you still want help with your fees, you will have to complete a new application. </p>
+        <p><strong>Reference: %{reference}</strong></p>
+        <p>We have received the additional information for your application for help with fees. However, there’s a problem with the documents you have sent.</p>
+        <p>(PLEASE EXPLAIN WHY EVIDENCE CAN'T BE ACCEPTED, FOR EXAMPLE: BANK STATEMENTS ARE OUT OF DATE</p>
+        <p>Unfortunately we’re unable to process your application and you will have to pay the full fee. This may have an impact on the original case or claim related to this application.</p>
+        <p>If you still want help with your fees, you will have to complete a new application.</p>
         <p>Yours sincerely,</p>
         <p>%{user_name}</p>
     part-payment:
       send_part_payment_html: |
         <p>Dear %{full_name}</p>
         <p><strong>Reference: %{reference}</strong></p>
-        <p>We have received your application for help with fees. You are eligible to make a part-payment and must pay %{part_payment_amount} towards the fee.</p>
+        <p>We have received your application for help with fees. You are eligible for some money off and must make a part-payment of %{part_payment_amount} towards the fee.</p>
         <p>We’ve calculated this as follows:</p>
         <p>Your total monthly income: %{total_income}</p>
         <p>Number of children: %{children}</p>
         <p>Amount to pay: %{part_payment_amount}</p>
         <p><strong>How to pay</strong></p>
         <ul class="list list-bullet">
-          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for the amount</li>
+          <li>by post with a cheque written out to ‘HM Courts and Tribunals Service’ for %{part_payment_amount}</li>
           <li>by debit card over the phone</li>
           <li>in person with a cheque, cash or debit card at the court counter</li>
         </ul>
-        <p>Please make sure you pay this amount by %{date_for_part_payment}</p>
+        <p>Please make sure you pay <strong>%{part_payment_amount}</strong> by <strong>%{date_for_part_payment}</strong>.</p>
         <p><strong>If you don’t agree with the amount you have to pay</strong></p>
-        <p>If you disagree with our decision about the amount you have to pay, you can appeal to the court or tribunal's Delivery Manager within 14 days of receiving this letter.</p>
+        <p>If you disagree with our decision about the amount you have to pay, you can appeal to %{name_delivery_manager}, Delivery Manager at %{court_tribunal_name}. You need to do this within 14 days of receiving this letter.</p>
```

Comment:
This was discussed with Leigh.  The delivery manager details are not available, we also cannot be sure of the specific court, as some appeals go to a regional delivery manager, not necessarily the same court as the current user.